example_deck.py

```diff
@@ -0,0 +1,47 @@
+try:
+    from typing import Annotated
+except ImportError:
+    from typing_extensions import Annotated
+
+import pandas as pd
+import plotly.express as px
+from flytekitplugins.deck.renderer import BoxRenderer, MarkdownRenderer
+
+import flytekit
+from flytekit import dynamic, task, workflow
+from flytekit.deck import default_deck
```

Comment:
I do not think we should make this global This is a problem in multi-threaded scenarios.

Reply:
I guess we can use contextvars?
cc @eapolinario / @wild-endeavor 

I did not even know about contextvars. Probably should replace FlyteContext?

Reply:
yeah can we just move this into ExecutionParameters for now? `flytekit.current_context().deck.

Why do we need multiple?  Can we just have the default one?  Or have people asked for multiple?